README.md

```diff
@@ -1,6 +1,6 @@
 # DC/OS Prometheus Alert Rules
 
-This repository is the home of the official DC/OS Prometheus alert rules.
+This repository is the home of the official DC/OS Prometheus, a sub-component of the DC/OS Monitoring package, alert rules.
```

Comment:
There is no `DC/OS Prometheus` component. This is a repository for the Prometheus Alert Rules for DC/OS. 

I would prefer that we explain the relationship with DC/OS Monitoring separately. This isn't really a sub-component. How about:

```suggestion
This repository is the home of the official DC/OS Prometheus alert rules. They are automatically included in releases of the DC/OS Monitoring framework. 
```